po/es.po

```diff
@@ -6,20 +6,20 @@ msgstr ""
 "PO-Revision-Date: 2021-09-14 15:25\n"
 "Last-Translator: \n"
 "Language-Team: Spanish\n"
-"Language: es_ES\n"
+"Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Crowdin-Project: clapper\n"
 "X-Crowdin-Project-ID: 473374\n"
-"X-Crowdin-Language: es-ES\n"
+"X-Crowdin-Language: es_ES\n"
 "X-Crowdin-File: /master/po/com.github.rafostar.Clapper.pot\n"
 "X-Crowdin-File-ID: 31\n"
 
 #: ui/clapper.ui:6
 msgid "Open Files..."
-msgstr "Abrir archivos"
+msgstr "Abrir archivo"
```

Comment:
This is changing the context of the string. Even if the file chooser on your mobile touchscreen device (Librem 5) does not allow selecting multiple files (I presume) you shouldn't assume that everyone in Spain uses Librem 5/Pinephone and not desktop where selecting multiple files is possible.